ultra/ultra/adapters/default_observation_image_adapter.py

```diff
@@ -0,0 +1,60 @@
+# MIT License
+#
+# Copyright (C) 2021. Huawei Technologies Co., Ltd. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+import gym
+import numpy as np
+
+from smarts.core.agent_interface import RGB
+from smarts.core.sensors import Observation
+
+
+_WIDTH = 64
+_HEIGHT = 64
+_RESOLUTION = 50 / 64
```

Comment:
Yes, I have seen the distortion as well. We can definitely change these values before this PR is merged. If I recall correctly, I think you said you had tried 84x84, but I also forget the resolution. I can try and see how larger values for the width and height look.

Also, note for myself: **Add comments to explain what these constants mean.**

Reply:
@AlexLewandowski, okay after looking at the World Models paper, I think going with 64x64 should be fine. They down-sampled Gym's "Car Racing" environment from a 600x400 image to a 64x64 image. Their down-sampled image quality is similar to our 64x64 image quality:

| Car Racing | SMARTS/ULTRA |
|-------------|-------------------|
| <img src="https://user-images.githubusercontent.com/49890535/120720288-0ea21900-c489-11eb-937a-2979e6222eda.png" width="256" height="256" /> | <img src="https://user-images.githubusercontent.com/49890535/120720562-87a17080-c489-11eb-9e0e-e42a74722755.png" width="256" height="256" /> |

Perhaps ours is even slightly more conducive to learning because we don't have all those different shades 🤔. Let me know your thoughts.

Reply:
The world models paper is model-based so they reconstruct the pixel observations. It would make sense that they would want a smaller image. Still, it's a good comparison and we should go ahead with this.